crates/rust-analyzer/src/to_proto.rs

```diff
@@ -433,14 +437,12 @@ pub(crate) fn inlay_hint(line_index: &LineIndex, inlay_hint: InlayHint) -> lsp_e
         },
         tooltip: None,
         padding_left: Some(match inlay_hint.kind {
-            InlayKind::TypeHint => false,
-            InlayKind::ParameterHint => false,
+            InlayKind::TypeHint | InlayKind::ParameterHint => false,
```

Comment:
We might have to adjust the padding a bit: with colons a type hint would render like `let x: i32 = 0;` (no padding), but without a colon it would look more like `let xi32 = 0`.

Reply:
Ye looks like we should do that